3.-Web-api-call-Microsoft-graph-for-personal-accounts/README.md

```diff
@@ -172,64 +165,28 @@ If you want to register your apps manually, as a first step you'll need to:
+What **differs from the previous chapter** is that you will use the same application ID for the client part as for the service part
 
-1. Navigate to the Microsoft identity platform for developers [App registrations](https://go.microsoft.com/fwlink/?linkid=2083908) page.
-1. Select **New registration**.
-1. When the **Register an application page** appears, enter your application's registration information:
-   - In the **Name** section, enter a meaningful application name that will be displayed to users of the app, for example `TodoListClient-v2`.
-   - Change **Supported account types** to **Accounts in any organizational directory and personal Microsoft accounts (e.g. Skype, Xbox, Outlook.com)**.
-   - Select **Register** to create the application.
 1. On the app **Overview** page, find the **Application (client) ID** value and record it for later. You'll need it to configure the Visual Studio configuration file for this project (`ida:ClientId` in `TodoListClient\App.Config`).
 1. From the app's Overview page, select the **Authentication** section.
-   1. In the **Redirect URIs** list, select for **TYPE** Public client (mobile & desktop). Then paste this value **urn:ietf:wg:oauth:2.0:oob** in the **REDIRECT URI** column. 
-   1. Select **Save**.
-1. Select the **API permissions** section
-   - Click the **Add a permission** button and then,
-   - Ensure that the **My APIs** tab is selected
-   - In the list of APIs, select the API `TodoListService-v2`.
-   - In the **Delegated permissions** section, ensure that the right permissions are checked: **access_as_user**. Use the search box if necessary.
-   - Select the **Add permissions** button
-
-> Important. Even if you declared the permission in the application registration to be (middletier)/**access_as_user**, in the code, you will need to acquire a token for (middletier)/**.default** in order to let the user consent to the downstream API (Graph) as well as the TodoListService. See [Configure the TodoListClient C# project](https://github.com/Azure-Samples/active-directory-dotnet-native-aspnetcore-v2/blob/master/2.%20Web%20API%20now%20calls%20Microsoft%20Graph/README.md#configure-the-todolistclient-c-project) below
-
-#### Register the client app as an authorized client application for the Web API
-
-In order for the user to be able to consent to the Web API and its downstream API, we need to register the client (TodolistClient) as an authorized
-application for the service. Here is how to do:
-
-1. Back in the application registration for the Web API (TodoListService):
-- In the list of pages for the app, select **Expose an API**
-- Select **Add a client application**
-- in the **ClientID** field, type the ClientID of the client app (TodoListClient)
-- In the authorized scope section, check the API permision you created for the service
-- Select **Add application**
+   - In the **Redirect URLs** | Add the following redirect URI: **urn:ietf:wg:oauth:2.0:oob**
```

Comment:
>**urn:ietf:wg:oauth:2.0:oob** [](start = 64, length = 29)

update to new redirectUri?

Reply:
I'll do a pass on the whole tutorial (I'd want the 3 folders to be consistent)